experiments/integrated/ozark/ozark.jl

```diff
@@ -229,42 +229,48 @@ sol = SciMLBase.solve(
 daily = sol.t ./ 3600 ./ 24
 savedir = joinpath(climalsm_dir, "experiments/integrated/ozark/")
 
+# This function will be used to compute averages over diurnal cycles. Input a
+# data series to average over and the total number of days in the data series, 
+# and it will return a vector of the average value at each timestamp in the 
+# day averaged over every day in the series.
+function diurnal_avg(series, days)
+    daily = floor(Int, length(series) / days)
+    daily_data = [
+        [sample for sample in series[i:1:(i + daily - 1)]] for
```

Comment:
would this work instead? 
`[series[i:1:(i + daily - 1)]] for i in 1:daily:(daily * days)]`

Reply:
Is there any risk that `daily` will correspond to slightly less than a day, and then over time the excerpt of the data we are considering as "a day" will shift in time?

How does one pass in the number of days if it is a fraction? e.g. if we have a non-integer number of days?

Perhaps renaming some of this could be clearer? `daily` -> `n_indices_per_day`, `daily_data` -> `data_by_day`? Im not sure what is best, but the current names dont clearly indicate what they are referring to.

Reply:
I cleaned up the naming conventions and neatened this code but we do still need to think about how we could do this if we had a non-integer number of days. For now it will work automatically without changing any of the args in `ozark.jl` for any integer number of days, but if we have a fractional number of days then the indexing no longer works for this method.